blocks/vip-locations/vip-locations.js

```
@@ -0,0 +1,63 @@
+export default async function decorate(block) {
+  // first row is link to plan
+  const planUrl = block.querySelector('a').href;
```

should we control if the author didn't setup a link to the map? I guess with this code it would raise a NPE

---

I think we can create a configuration where these links can be put. I need that for vip-areas API link as well. @chicharr Should any excel can work as config or there is a specific location (like .helix) where admins would put these configs. If its the first, then I will create one at the root.

---

@chicharr I can add a check , but what i have seen in Franklin projects sofar  is that if mandatory settings are missing, we let the block simply fail. The other blocks still get rendered. The author usally copies the block from some example page and if the author messes up , he will see it in preview , so it will never get published in a broken state.

@vdua in this case i don't think it makes much sense. Its a one time link to an asset not used anywhere else. So extracting it into a separate configuration makes the use of the block just more cumbersome.